.golangci.yml

```diff
@@ -77,8 +73,3 @@ linters-settings:
 issues:
   max-issues-per-linter: 0
   max-same-issues: 0
-
-  exclude:
```

Comment:
My intent here was to attempt to silence golangci-lints typecheck errors but it doesn't work :(

I wish they'd let them be silenced. Makes the editor quite frustrating when a package can't be built as random other packages get linted...

Reply:
Thank you for always reviewing so quickly!

> I wish they'd let them be silenced. Makes the editor quite frustrating when a package can't be built as random other packages get linted...

It seems like there are no typecheck errors in CI at the moment, but are you excluding them because they become annoying when running lint in the IDE during development?
If that's the case, I’m thinking of reverting the following commit.
[refactor: golangci-lint delete unnecessary issues](https://github.com/microsoft/typescript-go/pull/617/commits/deacd90165767a1619ea72cf68ff6934975db03a)

Reply:
Ah, I see. Since typecheck errors are compile-time errors, it’s annoying for each linter to warn about them even in the CI environment. 
cf. https://github.com/golangci/golangci-lint/issues/2912#issuecomment-1149995159

Got it! In that case, I’ll revert the  commit. 🙆

Reply:
Yeah, that's the one. There's probably no point in leaving the ignores if they don't work, though. I've had it work before, but probably it was years ago...